lib/tests/Makefile.am

```diff
@@ -3,6 +3,7 @@ lib_tests_TESTS		= \
 	lib/tests/test_lexer_block	\
 	lib/tests/test_cfg_tree		\
 	lib/tests/test_parse_number	\
+	lib/tests/test_generic_number	\
```

Comment:
test_generic_number is missing from lib/tests/CMakeLists.txt

Reply:
added, thanks.